src/middleware/logger/mod.rs

```diff
@@ -1,16 +1,6 @@
 //! Logging middleware.
 //!
-//! # Examples
-//!
-//! ```no_run
-//! # #[async_std::main]
-//! # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-//! let mut res = surf::get("https://httpbin.org/get")
-//!     .middleware(surf::middleware::logger::new())
-//!     .await?;
-//! dbg!(res.body_string().await?);
-//! # Ok(()) }
-//! ```
+//! This middleware is used by default unless the `"middleware-logger"` feature is disabled.
```

Comment:
I think it's still useful to show the example in case users disable the feature but want to use the logger for a particular client anyways?

Reply:
Fwiw it is shown below as `Logger::new()`